store/store.js

```diff
@@ -11,7 +11,7 @@ class Store {
   }
 
   request(...args) {
-    return this.options.request(...args)
+    return this.options.request.call(this, ...args)
```

Comment:
Sadly this doesn't cover table case. I am not sure why...

Reply:
Yes, I was thinking about tonight and I actually found the problem. This code binds to the current `this`, i.e. to each individual `Store` instance. Despite, we need to bind it to the certain `Root` instance.

I am adding a fix for this.